bin/fixmystreet.com/fixture

```diff
@@ -14,26 +14,96 @@ BEGIN {
     require "$d/../../setenv.pl";
 }
 
-use FixMyStreet::DB;
-
-my $body = FixMyStreet::DB->resultset("Body")->find_or_create({ name => 'Test City Council' });
-$body->body_areas->find_or_create({ area_id => 2504 });
-foreach ("Potholes", "Street lighting", "Graffiti") {
-    (my $email = lc $_) =~ s/ /-/g;
-    $body->contacts->find_or_create({
-        category => $_,
-        email => $email . '@example.net',
-        confirmed => 't',
-        deleted => 'f',
-        whenedited => \'current_timestamp',
-        editor => 'fixture',
-        note => 'Created by fixture'
-    });
+use FixMyStreet::Cobrand;
+use FixMyStreet::DB::Factories;
+use Getopt::Long::Descriptive;
+
+my ($opt, $usage) = describe_options(
+    '%c %o',
+    [ 'area_id=i', "MapIt area ID to create body for", { required => 1 } ],
+    [ 'empty', "Empty all tables of the database first" ],
+    [ 'dry-run', "Do not actually change the database" ],
```

Comment:
I'd prefer a safer option where, like `update-schema`, you need to provide a `--commit` flag in order for this script to do anything. Otherwise it'll just be a matter of time before I find some way to accidentally run this against fixmystreet.com 😱